apps/expand.go

```diff
@@ -48,6 +48,10 @@ type Expand struct {
 	// DisplayName, Name
 	Channel ExpandLevel `json:"channel,omitempty"`
 
+	// ChannelMember: expand model.ChannelMember if ChannelID and
+	// ActingUserID are set.
```

Comment:
Is there a difference between `all` and `summary`?

Reply:
no, it's a small-ish struct